test_helpers/main.go

```diff
@@ -148,6 +148,19 @@ func IsTarantoolVersionLess(majorMin uint64, minorMin uint64, patchMin uint64) (
 	return false, nil
 }
 
+// RestartTarantool restarts a tarantool instance for tests
+// with specifies parameters (refer to StartOpts)
+// which were specified in `inst` parameter.
+// `inst` is a tarantool instance that was started by
```

Comment:
We decided not to use apostrophes in documentation lines because it's not processed by godoc.

Reply:
Fixed.